src/reference/modules/administration/pages/performance.adoc

```diff
@@ -75,6 +75,32 @@ service restart rudder-jetty
 The amount of memory should be the half of the RAM of the server, rounded up to the nearest GB.
 For example, if the server has 5GB of RAM, 3GB should be allocated to Jetty.
 
+When RAM allocated to jetty reach 4 to 6GB (or higher), you may experience long freeze of Rudder, up 
+to several tens of seconds. In this is the case, you can change the JVM garbage collector 
+used to one better fitted for the kind of memory size:
+
+----
+
+# edit +/etc/default/rudder-jetty+ with your preferred text editor, for example vim:
+vim /etc/default/rudder-jetty
+
+Notice: that file is alike to +/opt/rudder/etc/rudder-jetty.conf+, which is the file with
+default values. +/opt/rudder/etc/rudder-jetty.conf+ should never be modified directly because
+modification would be erased by packaging in the following Rudder versuib update.
```

Comment:
in this file, in existing install, there is 
-XX:+CMSClassUnloadingEnabled
-XX:+UseConcMarkSweepGC


which one is taken into account then if we add the option below ?

Reply:
The parent ticket modified that file to use by default (ie if `JAVA_GC` is not defined) `CMSGC`, and to use whatever the user defined for it in the other case. 
See https://github.com/Normation/rudder-packages/pull/1971/

Reply:
you need to state that this solution is for 5.0.12 and later only then